Pipelines/YmlTemplates/postBuildSteps.yml

```diff
@@ -0,0 +1,35 @@
+# Shared post build template, run after the build template. Currently used by
+# - Domain API pipeline
```

Comment:
Is this a copy-paste issue or it's correctly mentioning Domain API pipeline?

Reply:
copy-paste